packages/@react-spectrum/s2/src/Tabs.tsx

```diff
@@ -137,15 +146,17 @@ const tablist = style({
       vertical: 12
     }
   },
+  flexShrink: 0,
+  flexBasis: '[0%]'
 });
 
 export function TabList<T extends object>(props: TabListProps<T>) {
-  let {density, isDisabled, disabledKeys, orientation = 'horizontal'} = useContext(TabsInternalContext);
+  let {density, isDisabled, disabledKeys, orientation} = useContext(TabsInternalContext);
```

Comment:
moved the default up to the top level so it's the same across everything and we don't need to remember to set the default in the future